### ISSUE TITLE
Super resolution fine-tuning 성능 저하

### ISSUE THREAD

**Issue by dcafplz** [playdata-aichief-ai/aichief-python-aiserver#4]
# 문제점
Fine-tuning 테스트 결과 오히려 기존 모델보다 성능이 저하됨.

# 예시
    
pretrain model    
![피보험자-성명-samsung-3](https://user-images.githubusercontent.com/105038597/197505556-4d27d4d2-19b1-4d99-a8c5-39c5c6f2fb45.jpg)

     
Fine-tuning model
![피보험자-성명-samsung-3](https://user-images.githubusercontent.com/105038597/197505598-2c1fcf8b-9be6-48f6-a199-7824f57c0f97.jpg)

# 추정원인
train dataset의 화질이 낮음(흔들림 등)
![img00006](https://user-images.githubusercontent.com/105038597/197505866-3b1d0e8d-28aa-470f-b6fe-2c15d77dd904.jpg)

새로운 데이터셋
![img1382097](https://user-images.githubusercontent.com/105038597/198922074-cb06f196-3ae2-4ab8-8cdf-ddffc93e84d3.jpg)

perceptual loss 미적용    
-> 적용완료

# to-do    

- [x] 새로운 학습 데이터셋 생성
- [x] perceptual loss 적용
- [x] 학습
- [x] 평가
